### ISSUE TITLE
go get go.etcd.io/etcd/clientv3 fails

### ISSUE THREAD

**Issue by lummie** [etcd-io/etcd#11361]
Expect `go get go.etcd.io/etcd/clientv3` to fetch dependencies correctly

`go get go.etcd.io/etcd/clientv3` result is:
```
go: finding github.com/coreos/go-systemd/journal latest
go: finding github.com/coreos/go-systemd latest
../../../go/pkg/mod/github.com/coreos/etcd@v3.3.17+incompatible/pkg/logutil/zap_journal.go:29:2: no matching versions for query "latest"
```

**Comment by lummie** [comment]
sorry, duplicate of #11345

**Comment by spzala** [comment]
> sorry, duplicate of #11345

Does the latest discussion there helpful to close this issue? Thanks!

**Comment by spzala** [comment]
This is fixed under #11557 I am closing the issue after successfully running the go get. Please reopen if needed. Thanks!

**Comment by tennessine** [comment]
It's duplicated, but I still put the right answer here for you, you're welcome.
add the code below to your go.mod file, below require block
```
replace github.com/coreos/go-systemd => github.com/coreos/go-systemd/v22 v22.0.0
```
